src/entity/Entity.ts

```diff
@@ -871,7 +867,7 @@ class Entity {
 
         if (this.parent && this._relativePosition) {
             let scd = this._getScaleByDistance();
-            pos = absolutCartesian.sub(this.parent.getAbsoluteCartesian()).scale(1 / scd);
+            pos = absolutCartesian.sub(this.parent.getAbsoluteCartesian()).scale(1 / scd).divA(this._absoluteScale);
```

Comment:
Just in case, it is good to read. As a proposal, we could try to make math simplification with less instructions.

Reply:
sometimes